presentations/AnalogL1/presentation.html

```diff
@@ -852,7 +852,7 @@
  - Highest I<sub>L(RMS)</sub> is at lowest V<sub>L(RMS)</sub> and highest VA specified (i.e., I<sub>L(RMS)</sub> = 7.5VA/13.6V = 0.55A<sub>RMS</sub>)
 - Using this information we can derive the relation
 \\[ P\_{is-loss} = I^2\_{L(RMS)} R\_s \quad \Rightarrow \quad 0.05 \geq 0.55^2 R\_s \quad \Rightarrow \quad R\_s \leq 165 m\Omega \\]
-- If R<sub>s</sub> is choosen as 160m&Omega;, V<sub>is</sub> measured at both highest and lowest i<sub>L</sub> are
+- If R<sub>s</sub> is chosen as 160m&Omega;, V<sub>is</sub> measured at both highest and lowest i<sub>L</sub> are
```

Comment:
> **Warning**  
> This may cause merge conflicts with #3.

To resolve this conflict, accept the changes from #3 and manually apply this typo fix by changing `choosen` -> `chosen`.